transform-plugins/src/main/java/co/cask/hydrator/plugin/ValueMapperLookupProvider.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.plugin;
+
+import co.cask.cdap.api.dataset.DatasetProperties;
+import co.cask.cdap.etl.api.LookupConfig;
+import co.cask.cdap.etl.api.LookupProvider;
+import co.cask.cdap.etl.api.LookupTableConfig;
+
+/**
+ * {@link LookupProvider} implementation for {@link ValueMapper}.
+ */
+public class ValueMapperLookupProvider {
```

Comment:
As per Sree's suggestion, JavaScriptTransform class has been referred which has the same design structure for this class.  We thought that it must be for future use and has to be followed in same way.
 Please confirm if we should keep or remove this class?